### PR TITLE
Add another plugin example using inline code

### PR DESCRIPTION
**Pull request by deinspanjer**

### CODE REVIEW

File: src/pages/postgraphile/plugins.md [hunk 1]
```diff
@@ -168,4 +168,27 @@ An example of a PostGraphile server plugin is [@graphile/operation-hooks](https:
...
+
+```js
+
+/**
+ * This plugin override changes the branding piece of graphiql.
+ */
+const graphiqlBrandingTweak = {
+    ['postgraphile:graphiql:html'](html) {
+        console.log("Applying GraphiQL Branding Tweak...");
+        return html.replace('i.a.createElement("div",null,"PostGraph",i.a.createElement("em",null,"i"),"QL")', '"GraphiQL for MyCompany"');
```
Comment (benjie) [inline src/pages/postgraphile/plugins.md]: This example is a little too brittle, it depends on the minified source code which could change for any number of reasons. Is there a more stable change you could make, for example adding a CSS file before `</head>`?

Comment (deinspanjer) [reply src/pages/postgraphile/plugins.md]: Yeah, I think we can use text transformation css for this example.  I'll work on that and update the PR.

Comment (benjie) [reply src/pages/postgraphile/plugins.md]: Thanks! 🙏

**Comment by benjie** [comment]
Thanks!

**Pull request merged**